nylas/models/drafts.py

```diff
@@ -0,0 +1,162 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import List, Optional, get_type_hints
+
+from dataclasses_json import dataclass_json
+from typing_extensions import TypedDict, NotRequired
+
+from nylas.models.attachments import CreateAttachmentRequest, Attachment
+from nylas.models.events import EmailName
+from nylas.models.list_query_params import ListQueryParams
+from nylas.models.messages import Message
+
+
+@dataclass_json
+@dataclass
+class Draft(Message):
+    """
+    A Draft object.
+
+    Attributes:
+        id (str): Globally unique object identifier.
+        grant_id (str): The grant that this message belongs to.
+        from_ (List[EmailName]): The sender of the message.
+        date (datetime): The date the message was received.
+        object: The type of object.
+        thread_id (Optional[str]): The thread that this message belongs to.
+        subject (Optional[str]): The subject of the message.
+        to (Optional[List[EmailName]]): The recipients of the message.
+        cc (Optional[List[EmailName]]): The CC recipients of the message.
+        bcc (Optional[List[EmailName]]): The BCC recipients of the message.
+        reply_to (Optional[List[EmailName]]): The reply-to recipients of the message.
+        unread (Optional[bool]): Whether the message is unread.
+        starred (Optional[bool]): Whether the message is starred.
+        snippet (Optional[str]): A snippet of the message body.
+        body (Optional[str]): The body of the message.
+        attachments (Optional[List[Attachment]]): The attachments on the message.
+        folders (Optional[List[str]]): The folders that the message is in.
+        created_at (Optional[int]): Unix timestamp of when the message was created.
+    """
+
+    object: str = "draft"
```

Comment:
should this on other object types too, like attachments? I see you added it to messages

Reply:
`attachments` doesn't need one because this `object` key typically only exists on top-level objects with getter endpoints. Like Drafts, Calendars, Messages, Events, etc. Just tested for attachment objects within messages and drafts, no object key is present.